tools/postinstall-android.js

```diff
@@ -105,7 +88,7 @@ if (data.indexOf("SalesforceHybrid") < 0)
     const oldAndroidDepTree = "android {";
     const newAndroidDepTree = "android {\n\tpackagingOptions {\n\t\texclude 'META-INF/LICENSE'\n\t\texclude 'META-INF/LICENSE.txt'\n\t\texclude 'META-INF/DEPENDENCIES'\n\t\texclude 'META-INF/NOTICE'\n\t}";
     replaceTextInFile(path.join(appProjectRoot, 'app', 'build.gradle'), oldAndroidDepTree, newAndroidDepTree);
-    const newLibDep = "api project(':SalesforceHybrid')";
+    const newLibDep = "api 'com.salesforce.mobilesdk:SalesforceHybrid'";
```

Comment:
Perhaps a boon for developer experience is that switching from the MSDK composite build to any hosted version someday is simply removing the `includeBuild` statement and giving this a version number.  I'm not certain when that might be useful, but it would be easier if it was!

Reply:
💫 That's exactly correct.  And, a version could be specified here on this line to provide a stable Maven Central build of MSDK SalesforceHybrid by default, where as it is here it will always pull the latest.

Another trick we have from LMR is that the `includeBuild` statement in `settings.gradle` can us an if-condition to see if the local MSDK clone/submodule is present.  If it is, use the composite build for fast dev integration.  If not, simply skip it and pull from Maven Central.

The nice thing about that is we get out of the business of programmatically editing the Gradle scripts with shell/shell.js.

Reply:
I remember that from the LMR repo. So then all we would is clone the repo in the expected place when in dev (that would happen in forcehybrid since we don't have a install.js in the template for hybrid templates). We would do the same for native templates (but in install.js) and react native template.